### DISCUSSION TITLE
Is there any presence members limit?

### DISCUSSION THREAD

**Discussion by Spharian** [beyondcode/laravel-websockets#1131] [Q&A]
Hi,

When I read the documentation of Pusher/Ably there are members limit (of max. 200 users) on the presence channels. I'd like to know if such a limit is in place with laravel-websockets?

Kind regards

**Comment by rachids** [comment]
Hello,

I haven't yet tried this package but [documentation](https://beyondco.de/docs/laravel-websockets/faq/deploying#open-connection-limit) states a limit of 1024 concurrent users by default.
